plugins/catalog-backend-module-gitlab/src/lib/types.ts

```diff
@@ -13,9 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
D:

i'm the type of person who keeps re-ADDING these newlines :) Not fond of things being squashed together with the license info